### PR TITLE
feat: Across Lite support in /limits and /suggested-fees endpoints

### PR DESCRIPTION
**Pull request by melisaguevara**
Required changes in /limits and /suggested-fees endpoints to support Across Lite.

/limits: when the requested route involves a lite chain, we avoid including the LP balances when defining limits.
/suggested-fees: the LP fee must be 0 when a route involves a lite chain. To achieve that we conditionally overwrite the rate model key we query.

### CODE REVIEW

**Review by nicholaspai — APPROVED** [review]
This makes sense to me! Would like dong ha to review also

**Review by dohaki — COMMENTED** [review]
LGTM! But you might need to rebase/merge and fix some conflicts because I had to change some minor things in https://github.com/across-protocol/frontend-v3/pull/1077

File: api/limits.ts [hunk 1]
```diff
@@ -88,16 +90,32 @@ const handler = async (
...
+        functionName: "globalConfig",
+        args: [encodedLiteChainsKey],
+      },
     ];
 
     const [tokenPriceNative, _tokenPriceUsd] = await Promise.all([
       getCachedTokenPrice(
         l1Token.address,
-        sdkUtils.getNativeTokenSymbol(destinationChainId).toLowerCase()
+        sdk.utils.getNativeTokenSymbol(destinationChainId).toLowerCase()
```
Comment (james-a-morris) [inline api/limits.ts:118]: Was this change related to the recent bundle data size reduction work by @dohaki

**Review by james-a-morris — COMMENTED** [review]
One question otherwise lgtm

File: api/limits.ts (hunk 1 above)
Comment (dohaki) [reply api/limits.ts:118]: No, the FE bundle size reduction is independent of the code of our serverless functions in `/api/*`

**Review by amateima — APPROVED** [review]

File: api/limits.ts [hunk 2]
```diff
@@ -149,17 +167,24 @@ const handler = async (
       ),
     ]);
 
-    let { liquidReserves } = hubPool.interface.decodeFunctionResult(
-      "pooledTokens",
-      multicallOutput[1]
+    let { liquidReserves } = multicallOutput[1];
+    const [liteChainIdsEncoded] = multicallOutput[2];
+    const liteChainIds =
+      liteChainIdsEncoded === "" ? [] : JSON.parse(liteChainIdsEncoded);
```
[the remark this answers is not in the document]
Comment (melisaguevara) [reply api/limits.ts:173]: Yes, it has the quotes but it is not needed to remove them. JSON.parse handles that.

**Comment by melisaguevara** [comment]
> LGTM! But you might need to rebase/merge and fix some conflicts because I had to change some minor things in #1077

@dohaki I've rebased again, would you please review again? just in case

**Review by dohaki — APPROVED** [review]
LGTM 👍

File: api/limits.ts (hunk 1 above)
Comment (melisaguevara) [reply api/limits.ts:118]: > If u refer to this specific change, this is just because i renamed the import of the sdk

**Comment by dohaki** [comment]
@melisaguevara sorry we had to merge some urgent changes again here https://github.com/across-protocol/frontend-v3/pull/1080... So you might need to rebase again 😓

**Pull request merged**